DOCSUP-17856: [CLICKHOUSEDOCS] Document the merge_tree_min_rows_for_c… (#32924)
ClickHouse/ClickHouse: docs/en/operations/settings/settings.md

Possible values:

-   Positive integer.

Default value: `163840`.

## merge_tree_min_rows_for_concurrent_read_for_remote_filesystem {#merge-tree-min-rows-for-concurrent-read-for-remote-filesystem}

> If at least as many lines are read from one file, the [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md) engine can reads parallelized, when reading from remote filesystem.

Reviewer: ```suggestion
The minimum number of lines that are read from one file before [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md) engine can parallelize reading, when reading from remote filesystem.
```

Possible values:

-   Positive integer.

Default value: `163840`.

## merge_tree_min_bytes_for_concurrent_read {#setting-merge-tree-min-bytes-for-concurrent-read}

If the number of bytes to read from one file of a [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md)-engine table exceeds `merge_tree_min_bytes_for_concurrent_read`, then ClickHouse tries to concurrently read from this file in several threads.

Possible value:

-   Positive integer.

Default value: `251658240`.

## merge_tree_min_bytes_for_concurrent_read_for_remote_filesystem {#merge-tree-min-bytes-for-concurrent-read-for-remote-filesystem}

> If at least as many bytes are read from one file, the [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md) engine can reads parallelized, when reading from remote filesystem.

Reviewer: ```suggestion
The minimum number of bytes to read from one file before [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md) engine can parallelize reading, when reading from remote filesystem.
```
